scripts/scripts.js

```diff
@@ -278,6 +301,22 @@ function loadDelayed() {
   import('./sidekick.js').then(({ initSidekick }) => initSidekick());
 }
 
+/**
+ * Fetch filtered search results
+ * @returns List of search results
+ */
+export async function fetchSearch() {
+  window.searchData = window.searchData || {};
+  if (Object.keys(window.searchData).length === 0) {
+    const lang = document.documentElement.lang;
```

Comment:
Please use getLanguage() method of utils.js

Reply:
sure